mmcv/ops/csrc/common/spconv/tensorview/helper_kernel.cu.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+namespace tv {
+namespace detail {
+
+template <typename T>
+class KernelLoop {
```

Comment:
Shall we use "KERNEL_LOOP" which is in `common_cuda_helper` or move these "utils" to `common_cuda_helper`?

Reply:
Sorry about that I cannot find `KERNEL_LOOP` in `common_cuda_helper`, or the `KERNEL_LOOP` may exist in another file. Could you please give me more details, like location, about `KERNEL_LOOP`?

Reply:
I mean [this](https://github.com/open-mmlab/mmcv/blob/master/mmcv/ops/csrc/common/cuda/common_cuda_helper.hpp#L6). It seems that `class KernelLoop` is more general, shall we move it to `common_cuda_helper`?